tests/cli/test_cloud_execution.py

```diff
@@ -46,6 +46,32 @@ def test_cloud_status(runid):
     assert result.exit_code == 0
 
 
+def test_cloud_status_finish_with_no_task_run(monkeypatch, runid):
+
+    class MockResponse:
+        exit_code = 0
+
+        @staticmethod
+        def output():
+            return 'Pipeline finished... \n\
+                Pipeline finished due to no newly triggered tasks, \
+                    try build with --force'
+
+    def mock_status(*args, **kwargs):
+        return MockResponse()
+
+    monkeypatch.setattr(CliRunner, 'invoke', mock_status)
+    runner = CliRunner()
+    result = runner.invoke(cli, ['cloud', 'status', runid])
+
+    assert 'taskid' not in result.output()
```

Comment:
The assertions here are off, what we're looking to test is the user output in the switchcase, make sure to use the capsys module and read the out pipe and assert on it. No need to assert stuff on the mock since it doesn't test anything.